PrototipoP31P2021/src/vista/MantenimientoAlumnos.java

```diff
@@ -0,0 +1,464 @@
+          
+
+
+            alumnosdao.insert(alumnos);
+
+            try {
+                llenar_tabla();
+            } catch (SQLException ex) {
+                Logger.getLogger(MantenimientoAlumnos.class.getName()).log(Level.SEVERE, null, ex);
+            }
+        } else {
+            JOptionPane.showMessageDialog(null, "Hay campos vacíos", "Error", JOptionPane.ERROR_MESSAGE);
+        }
+    }//GEN-LAST:event_btnRegistrarActionPerformed
+
+    private void btnBuscarActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnBuscarActionPerformed
+        // TODO add your handling code here:
+     Alumnos alumnos = new Alumnos();
+        AlumnossDAO alumnosdao = new AlumnossDAO();
+        alumnos.setCarnet_alumno(txtbuscado.getText());
+
+        alumnos = alumnosdao.query(alumnos);
+
+        Txtcarnet.setText(alumnos.getCarnet_alumno());
+        TxtNombre.setText(alumnos.getNombre_alumno());
+         TxtDirrecion.setText(alumnos.getDireccion_alumno());
+        TxtTelefono.setText(alumnos.getTelefono_alumno());
+         TxtEmail.setText(alumnos.getEmail_alumno());
+        TxtEstatus.setText(alumnos.getEstatus_alumno());
+      
+    }//GEN-LAST:event_btnBuscarActionPerformed
+
+    private void btnModificarActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnModificarActionPerformed
+     Alumnos alumnos = new Alumnos();
+        AlumnossDAO alumnosdao = new AlumnossDAO();
+       if (Txtcarnet.getText().length() != 0 && TxtNombre.getText().length() != 0
+                && TxtDirrecion.getText().length() != 0&& TxtTelefono.getText().length() != 0
+                && TxtEmail.getText().length() != 0&& TxtEstatus.getText().length() != 0) {
+            String carnet_alumno = Txtcarnet.getText();
+            String nombre_alumno = TxtNombre.getText();
+             String direccion_alumno = TxtDirrecion.getText();
+            String telefono_alumno = TxtTelefono.getText();
+              String email_alumno = TxtEmail.getText();
+            String estatus_alumno = TxtEstatus.getText();
+            
+
+            alumnos.setCarnet_alumno(carnet_alumno);
+            alumnos.setNombre_alumno(nombre_alumno);
+            alumnos.setDireccion_alumno(direccion_alumno);
+            alumnos.setTelefono_alumno(telefono_alumno);
+            alumnos.setEmail_alumno(email_alumno);
+            alumnos.setEstatus_alumno(estatus_alumno);
+
+    
+            try {
+                llenar_tabla();
+            } catch (SQLException ex) {
+                Logger.getLogger(MantenimientoAlumnos.class.getName()).log(Level.SEVERE, null, ex);
+            }
+        } else {
+            JOptionPane.showMessageDialog(null, "Hay campos vacíos", "Error", JOptionPane.ERROR_MESSAGE);
+        }
+    }//GEN-LAST:event_btnModificarActionPerformed
+
+    private void TxtTelefonoActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_TxtTelefonoActionPerformed
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.